### ISSUE TITLE
Loss skyrockets randomly, cause and effect unkno

### ISSUE THREAD

**Issue by npeirson** [tensorflow/models#6007]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. Really, they do. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow, per-say.

I left your pre-written message here so you could be as annoyed with it as I am.  

------------------------

### System information
- **What is the top-level directory of the model you are using**: `models/research/`
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Debian 9 nightly
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: nightly
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: nightly
- **GPU model and memory**: Tesla K80
- **Exact command to reproduce**: follow your own "pet detector" tutorial. <-- seriously, just do this

You can collect some of this information using our environment capture script:

### Describe the problem
Loss leaps occasionally, scheduled learning rates shouldn't permit such dramatic changes. Not sure that they're actually causing a problem, but want to get your insight on what's going on here. Happens every ~10,000 steps, started after ~25k steps. Causes?

### Source code / logs
```
 master-replica-0 loss = 0.049381822, step = 62104 (56.439 sec) I  master-replica-0
 master-replica-0 loss = 0.20843819, step = 62204 (57.136 sec) I  master-replica-0
 master-replica-0 loss = 0.08859253, step = 62304 (56.564 sec) I  master-replica-0
 master-replica-0 loss = 0.14603062, step = 62404 (56.244 sec) I  master-replica-0
 master-replica-0 loss = 5676524600000000.0, step = 62504 (56.958 sec) I  master-replica-0
 master-replica-0 loss = 0.083275326, step = 62604 (56.345 sec) master-replica-0 
```

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
